src/SnmpLibrary/library.py

```diff
@@ -307,7 +311,8 @@ def _set(self, *oid_values):
             self._active_connection.cmd_gen.setCmd(
                 self._active_connection.authentication_data,
                 self._active_connection.transport_target,
-                *oid_values
+                *oid_values,
+                contextName=self._active_connection.context_name,
```

Comment:
please remove the ',' from the end of the line

Reply:
Done in 0f18d150a6ac21aaf7dcf4dfddbf732bcacc3ace